Formula/e2fsprogs.rb

```diff
@@ -37,8 +37,15 @@ def install
     # Enforce MKDIR_P to work around a configure bug
     # see https://github.com/Homebrew/homebrew-core/pull/35339
     # and https://sourceforge.net/p/e2fsprogs/discussion/7053/thread/edec6de279/
-    system "./configure", "--prefix=#{prefix}", "--disable-e2initrd-helper",
-                          "MKDIR_P=mkdir -p"
+    args = [
+      "--prefix=#{prefix}",
+      "--disable-e2initrd-helper",
+      "MKDIR_P=mkdir -p",
+    ]
+    on_linux do
+      args << "--enable-elf-shlibs"
```

Comment:
What happens with `--enable-bsd-shlibs` on macOS?

Reply:
Tested locally. Seems to work. Might need `ENV.deparallelize` for `make install`.

Reply:
I got an error when trying it on Mac (something about a missing soname). I'll try a new run on CI to see what happens.

Reply:
>2021-05-28T10:25:07.8283120Z ld: unknown option: -soname
 2021-05-28T10:25:07.8284400Z clang: error: linker command failed with exit code 1 (use -v to see invocation)

Reply:
Still hitting the error. I added `ENV.deparallelize` but I don't think it's related to this error. You said you were able to build this on Mac?